client/src/components/layout/ReviewDeleteButton.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+
+const ReviewDeleteButton = ({ user, review, getPlanet }) => {
+  
+  const deleteReview = async (event) => {
+    event.preventDefault()
+    const reviewBody = {
+      reviewId: review.id,
+    }
+    try {
+      const response = await fetch(`/api/v1/planets/${review.planetId}/reviews/delete-review`, {
+        method: "DELETE",
+        headers: new Headers({
+          "Content-Type": "application/json",
+        }),
+        body: JSON.stringify(reviewBody),
+      })
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw (error)
+      }
+      getPlanet()
```

Comment:
No. This is cheating! 

refetching means more DB queries and fetches. If you need to remove the review from your state, do it with a filter method